### DISCUSSION TITLE
Plausible proxy using Express server

### DISCUSSION THREAD

**Discussion by yathomasi** [plausible/analytics#1876] [Anything Else]
We are using `Express server` to proxy the `script` and `event` which are not on [the list](https://plausible.io/docs/proxy/introduction). 

The script is on `/api/.../script.js` and the event API on `/api/event`. Although it's working(the script is loaded and the event request is `ok`) there is a decrease in traffic in a significant amount.

Is there something we are missing? 
Does the path matters for `script.js`?

**Comment by ukutaht** [comment]
Is your proxy server blocked by uBlock origin? Did you use a name like `analytics` or `tracking` in the domain name?

**Reply by yathomasi** [reply]
No, it's not blocked by uBlock orign. From my testing, the script is loaded fine and the event `POST` request also has 202 `Ok` response.

**Reply by ukutaht** [reply]
OK then I'm not sure what it could be. You should see an increase of traffic when using a proxy vs not. It's difficult to say from our end why the events are not arriving on our side.
